lib/api/controllers/funnelController.js

```
@@ -153,11 +173,11 @@ function FunnelController (kuzzle) {
           errorMessage = errorMessage.toLowerCase().replace(/[^a-zA-Z0-9-_]/g, '-').replace(/[-]+/g, '-').split('-');
           errorMessage = errorMessage.filter(value => value !== '').join('-');
 
-          request.data.body.sufix = 'handled-'.concat(errorType.toLowerCase(), '-', errorMessage);
+          request.data.body.suffix = 'handled-'.concat(errorType.toLowerCase(), '-', errorMessage);
```

Object.seal() does not cause exception throws. It only fails silently.
Fixed.